frontend/src/app/testResults/TestResultPrintModal.tsx

```diff
@@ -1,58 +1,24 @@
 import React from "react";
-import { gql } from "@apollo/client";
 import Modal from "react-modal";
 import classnames from "classnames";
 import { Trans, useTranslation } from "react-i18next";
 
 import Button from "../commonComponents/Button/Button";
+import CovidResultGuidance from "../commonComponents/CovidResultGuidance";
 import { displayFullName } from "../utils";
 import "./TestResultPrintModal.scss";
 import logo from "../../img/simplereport-logo-black.svg";
 import { QueryWrapper } from "../commonComponents/QueryWrapper";
 import LanguageToggler from "../../patientApp/LanguageToggler";
 import { formatDateWithTimeOption } from "../utils/date";
-
-export const testQuery = gql`
-  query getTestResultForPrint($id: ID!) {
-    testResult(id: $id) {
-      dateTested
-      result
-      correctionStatus
-      deviceType {
-        name
-        model
-      }
-      patient {
-        firstName
-        middleName
-        lastName
-        birthDate
-      }
-      facility {
-        name
-        cliaNumber
-        phone
-        street
-        streetTwo
-        city
-        state
-        zipCode
-        orderingProvider {
-          firstName
-          middleName
-          lastName
-          NPI
-        }
-      }
-    }
-  }
-`;
+import { GetTestResultForPrintDocument } from "../../generated/graphql";
 
 interface StaticTestResultModalProps {
   testResultId: string | undefined;
   testResult: any;
```

Comment:
nit: Could we update the type that we're using here to match the MultiplexResult interface we're defining in `QueueItem.tsx`? Using any in the places where it's applied turns off type checking, and I don't want a malformed type issue to cause the modal to display incorrectly (especially since the way we're gating certain parts of the UI on 43-56 is assuming the type has a certain shape). Similar thought applies throughout the rest of this file too.

Reply:
Ok! Thank you, Bob! Will do 😸

Reply:
@fzhao99 Ok updated the types throughout the file! Let me know if you see anything I can improve. Thank you!! 😸